schedule/yast/maintenance/autoyast_create_hdd_maintenance_deregister_aarch64.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: autoyast_create_hdd_maintenance_aarch64
+AUTOYAST: autoyast_sle15/autoyast_scc_up.xml
+schedule:
+  - autoyast/prepare_profile
+  - installation/bootloader_start
+  - autoyast/installation
+  - installation/first_boot
+  - console/system_prepare
+  - qa_automation/patch_and_reboot
```

Comment:
For our cases of creating images by autoyast installation, no need to do patch here, it is already fresh install.